CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
```

Comment:
Please keep the Change log in the GitHub releases to have a single source of truth.

Reply:
✅